src/bundle/ui-dev/src/modules/universal-discovery/components/selected-items/selected.items.panel.js

```diff
@@ -0,0 +1,149 @@
+import React, { useContext, useState, useEffect, useRef, useMemo } from 'react';
+
+import {
+    parse as parseTooltip,
+    hideAll as hideAllTooltips,
+} from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/tooltips.helper';
+import {
+    getBootstrap,
+    getAdminUiConfig,
+    getTranslator,
+} from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/context.helper';
+
+import Icon from '../../../common/icon/icon';
+import { createCssClassNames } from '../../../common/helpers/css.class.names';
+
+import { AllowConfirmationContext, SelectedItemsContext } from '../../universal.discovery.module';
+import { CLEAR_SELECTED_ITEMS } from '../../hooks/useSelectedItemsReducer';
+
+const SelectedItemsPanel = () => {
+    const Translator = getTranslator();
+    const adminUiConfig = getAdminUiConfig();
+    const itemsComponentsMap = useMemo(() => {
+        const { universalSelectItemsComponentsConfigs } = adminUiConfig.universalDiscoveryWidget;
```

Comment:
We already have `selectedItemActions` config for selected locations, so I've chosen `universalSelectItemsComponentsConfigs` but I'm not happy about this name. Better name suggestions are welcomed.

Reply:
I tried to come up with a proposal:
universalSelectionComponentsConfig - Barely shorter and not even sure if better